### ISSUE TITLE
Logging of hidden qualities prefers id over qualityId

### ISSUE THREAD

**Issue by kkotenko** [lensvol/fl-quality-lantern#3]
**Setup**
Here is a partial response to a request to `/storylet` (particularly "Approaching the Lifeberg's Hunting Grounds"):

```json
{
	"qualityRequirements": [
		{
			"allowedOn": "Character",
			"qualityId": 142286,
			"qualityName": "Zailing Route",
			"tooltip": "You unlocked this with <span class='quality-name'>Zailing Route</span> 3<em> (you needed exactly 3)</em>",
			"category": "Hidden",
			"nature": "Status",
			"status": "Unlocked",
			"isCost": false,
			"image": "evade2",
			"id": 201395
		},

	]
}
```
**Observed behavior:**
The extension logged the value of `id` to the console:

`[FL Quality Lantern] Hidden quality: Zailing Route  (ID: 201395)`

**Expected behavior:**
The extension logs the value of `qualityId` to the console.
`[FL Quality Lantern] Hidden quality: Zailing Route  (ID: 142286)`

**Additional information:**
From what I understand, id 201395 is the ID of the primary key of the database entry which connects the storylet "Approaching the Lifeberg's Hunting Grounds" to the quality Zailing Route. "Zailing Route" has ID 142286.

I am not sure when FL chooses to serve a JSON which contains the id of the property in "id" or "qualityId". I _believe_ that if `qualityId` is present, it is always to be preferred over `id`.

**Comment by lensvol** [comment]
Thanks, will fix it.

**Comment by lensvol** [comment]
Fixed in 17b8b521e6b122b4933528f76ef281782a33b9eb
